packages/styles/scss/components/masthead/_masthead.scss

```diff
@@ -245,6 +248,12 @@ $search-transition-timing: 95ms;
       &--active {
         border: 2px solid $interactive-01;
         background-color: $ui-01;
+
+        @include carbon--breakpoint-up(sm) {
+          position: absolute;
+          top: 0;
+          z-index: 6001;
```

Comment:
@kennylam 6001?

Reply:
@jeffchew Yes, styles have changed a bit to allow the overlay/nav to cover the L1 in that breakpoint. Default carbon z-index is 6000 and the close button gets covered. So it's now 6001